modules/aws-outposts-machine-set.adoc

```diff
@@ -0,0 +1,216 @@
+--
+
+. Configure the new compute machine set to create edge compute machines in the Outpost by editing the following values in the `<new_machine_set_name_1>.yaml` file:
++
+--
+.Example compute machine set for AWS Outposts
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> # <1>
+  name: <infrastructure_id>-outposts-<availability_zone> # <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-outposts-<availability_zone>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: outposts
+        machine.openshift.io/cluster-api-machine-type: outposts
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-outposts-<availability_zone>
+    spec:
+      metadata: {}
+      providerSpec:
+        value:
+          ami:
+            id: <ami_id> # <3>
+          apiVersion: machine.openshift.io/v1beta1
+          blockDevices:
+            - ebs:
+                volumeSize: 120
+                volumeType: gp2 # <4>
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: <infrastructure_id>-outposts-profile
+          instanceType: m5.xlarge # <5>
+          kind: AWSMachineProviderConfig
+          placement:
+            availabilityZone: <availability_zone>
+            region: <region> # <6>
+          securityGroups:
+            - filters:
+              - name: tag:Name
+                values:
+                  - <infrastructure_id>-outposts-sg
+          subnet:
+            id: <subnet_id> # <7>
+          tags:
+            - name: kubernetes.io/cluster/<infrastructure_id>
+              value: owned
+          userDataSecret:
+            name: worker-user-data
+      taints: # <8>
+        - key: node-role.kubernetes.io/outposts
+          effect: NoSchedule
+----
+<1> Specifies the cluster infrastructure ID.
+<2> Specifies the name of the compute machine set. The name is composed of the cluster infrastructure ID, the `outposts` role name, and the Outpost availability zone.
+<3> Specifies the Amazon Machine Image (AMI) ID.
+<4> Specifies the EBS volume type. AWS Outposts requires gp2 volumes.
+<5> Specifies the AWS instance type. You must use an instance type that is configured in your Outpost.
+<6> Specifies the AWS region in which the Outpost availability zone exists.
+<7> Specifies the dedicated subnet for your Outpost.
+<8> Specifies a taint to prevent user workloads from being scheduled on nodes that have the `outposts` role.
```

Comment:
maybe this?
```suggestion
<8> Specifies a taint to prevent user workloads from being scheduled on nodes that have the label `node-role.kubernetes.io/outposts` role.
```